students/ejackie/session03/mailroom.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python
+
+##################################################
+## Author: Jackie Cheung
+## Date: 2018/10/23
+## Version: 0.1
+## Class: Au2018-Py210B
+## Description: Mailroom Part 1 - sending Thank You email and create report
+##################################################
+
+import sys  # imports go at the top of the file
+
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ]
+
+prompt = "\n".join(("Welcome to the Mailroom!",
+          "Please choose from below options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Exit",
+          ">>> "))
+
+# Looking up the donor and add them to database if not exist
+def send_a_thank_you(donor_db):
+    name = input("Please enter the full name of the donor: ")
+    if(name == "list"):
```

Comment:
Minor: No need to put `(`, `)` around Python's `if`/`while` conditions (and it's not recommended).

Reply:
Thanks for pointing that out! Removed ( ) from the if statements.